addons/web/static/tests/views/kanban_tests.js

```diff
@@ -2054,6 +2054,45 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('quick create record if grouped on date(time) field with attribute can_use_group_value: true',
```

Comment:
Could be great to check that field has value coming from "server" (aka, the mock), if field is present in quick create template. Even if mock server could be de-synchronized from real server, at least we check the we have something that makes sense.

To check if `quick_create_view="crm.quick_create_opportunity_form"` works in unit tests, in that case we could specify a small form with date(time) field and check that we have a value in that field.